server/resp/src/main/java/org/infinispan/server/resp/logging/Log.java

```diff
@@ -20,6 +21,9 @@
  */
 @MessageLogger(projectCode = "ISPN")
 public interface Log extends BasicLogger {
+   Log CONFIG = Logger.getMessageLogger(Log.class, "org.infinispan.CONFIG");
```

Comment:
Can't we just reference the public ones on the other Log interfaces?

Reply:
Those are instances for different interfaces